src/webview/view/components/forms/spfxProject/GeneralInfoStep.tsx

```diff
@@ -104,9 +109,7 @@ export const GeneralInfoStep: React.FunctionComponent<IGeneralInfoProps> = ({
                     </>
                 }
                 <div className={'mb-2'}>
-                    <label className={'block mb-1'}>
-                        What component you wish to create?
-                    </label>
+                    <LabelWithTooltip label={"Component Type"} tooltip={getComponentType ? getComponentType.description : ""} />
```

Comment:
lets keep the question label for that. What do you think?

```suggestion
                    <LabelWithTooltip label={"What component you wish to create?"} tooltip={getComponentType ? getComponentType.description : ""} />
```